db/migrate/20220502133047_add_de_feature_flags.rb

```diff
@@ -0,0 +1,23 @@
+class AddDeFeatureFlags < Mongoid::Migration
+  # mirror of FeatureFlag.rb, so this migration won't error if that class is renamed/altered
+  class FeatureFlagMigrator
```

Comment:
I understand why we had this originally, but at this point we're never going to rename the `feature_flags` collection/model, so this is unnecessary.

Reply:
Makes sense, removed in 743a1bf.